types/material-colors/index.d.ts

```diff
@@ -0,0 +1,59 @@
+// Type definitions for materialColors 1.2
+// Project: https://github.com/shuhei/materialColors
+// Definitions by: Noah Overcash <https://github.com/ncovercash>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export as namespace material;
```

Comment:
I think this should be `materialColors`: <https://github.com/shuhei/material-colors/blob/ee408d4d6f0ecd0bb7f467696ca0e43ec1736711/dist/colors.js#L7>

```suggestion
export as namespace materialColors;
```